bemlint/lib/tests/element-inside-parent-block.test.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const rule = require('../rules/element-inside-parent-block');
+const { makeLinter } = require('../lint/makeLinter');
+const defaultConfig = require('../defaultConfig');
+
+describe(`Test ${rule.id} rule:`, () => {
+  const config = {
+    ...defaultConfig,
+    rules: {
+      ...defaultConfig.rules,
+      [rule.id]: true,
+    },
+  };
```

Comment:
```suggestion
  const config = {
    rules: {
      [rule.id]: true,
    },
  };
```

I believe it is enough